appyters/Reactome_Harmonizome_ETL/Reactome.ipynb

```diff
@@ -102,30 +92,29 @@
    "outputs": [],
    "source": [
     "%%appyter code_eval\n",
-    "\n",
     "{% do DescriptionField(\n",
-    "    name='description',\n",
-    "    text='The example below was sourced from <a href=\"http://reactome.org/pages/download-data/\" target=\"_blank\">reactome.org</a>. If clicking on the example does not work, it should be downloaded directly from the source.',\n",
+    "    name= 'Description',\n",
+    "    text= 'The example below was sourced from <a href=\"http://reactome.org/pages/download=data/\" target =\"_blank\">reactome.org</a>. If clicking on the example does not work, it should be downloaded directly from the source.',\n",
     "    section='data'\n",
-    ") %}\n",
+    ")%}\n",
     "\n",
     "{% set df_file = FileField(\n",
-    "    constraint='.*\\.zip$',\n",
-    "    name='pathways_gene', \n",
-    "    label='Pathways Gene Set (gmt.zip)', \n",
-    "    default='ReactomePathways.gmt.zip',\n",
-    "    examples={\n",
+    "    constrant= '.*/.gmt.zip$',\n",
```

Comment:
This constraint is not ideal, the filename should not require a slash in it. Consider: ``'.*\.gmt(\.zip)?'` instead.